high_card.rb

```diff
@@ -1,5 +1,45 @@
+require 'pry'
+
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K","A" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
+
+deck = []
+ranks.each_with_index do |rank, index|
+  suits.each do |suit|
+  deck << [rank,suit]
```

Comment:
what if each `card` was a hash instead of an array? Might make it easier to reference / add more key-values  such as "worth" to help when sorting